packages/core/src/types/oas3_1.ts

```diff
@@ -265,6 +266,17 @@ const SecurityScheme: NodeType = {
   extensionsPrefix: 'x-',
 };
 
+const DependentRequired: NodeType = {
+  properties: {},
+  additionalProperties: (value: any) => {
+    if (Array.isArray(value)) {
+      return { type: 'array', items: { type: 'string'}}
+    } else {
+      return
+    }
+  }
```

Comment:
It could be simplified like this:

```suggestion
  additionalProperties: { type: 'array', items: { type: 'string' } },
```